packages/next/package.json

```diff
@@ -33,6 +33,6 @@
   },
   "license": "SIL OPEN FONT LICENSE",
   "dependencies": {
-    "@next/font": "^13.5.6"
+    "next": "^14.0.1"
```

Comment:
I think we could use peer dependency as usually they require to run with nextjs?

```
peer dep {
   next: ^13.2 || ^14
}
```